### ISSUE TITLE
Get All doesn't work

### ISSUE THREAD

**Issue by thany** [Bionus/imgbrd-grabber#2078]
**Bug description**

Clicking the button **Get All** after having done a search, doesn't start any downloads.

**Steps to reproduce**

1. Set source to derpibooru
2. Search for `pony AND -safe AND -monochrome`
3. Click on **Get All**

**Expected behavior**

It should start downloading.

**Context**

As for the log, these are the last few entries after having tried to above several times:
```
[19:25:52.196][Info] [derpibooru.org][Json] Receiving page https://derpibooru.org/api/v1/json/search/images?per_page=20&page=1&q=pony AND -safe AND -monochrome
[19:25:52.208][Info] [derpibooru.org][Json] Parsed page https://derpibooru.org/api/v1/json/search/images?per_page=20&page=1&q=pony AND -safe AND -monochrome: 20 images (20), 0 tags (296), 34845 total (34845), 1743 pages (1743)
```
(anything before that is just looking around, nothing special)
In other words, clicking the button adds nothing to the log.

It may also help to know, after clicking the button I see 0% cpu utilization and 0% network utilization for Grabber.

As for settings, I left them default. I haven't touched any setting since I started using this tool, which is about a week ago.

**System information**

- OS: Windows 10
- Grabber version: 7.3.2

**Additional context**

A few days ago, downloading a large amount of images, similar to this amount, worked fine for danbooru.

**Comment by Bionus** [comment]
The "get all" button does not start downloads.
It only adds them to the "Downloads" tab where you can start them.
If you go there, can you see your added download and start it with the buttons at the bottom / right clicking it?

**Comment by thany** [comment]
Hm, you're right. Thanks!
I don't know how I managed to do it last time then. Just messing around until something happened in a tool I'm not familiar with 🤷‍♂️

Can you maybe add some ind of feedback that makes it clear what actually happens when clicking the button? Because as you might know, clicking the button *appears* to do nothing, as nothing happens visually. Do you think you can come up with a UX-friendly thing to provide a visual cue?

**Comment by FichteFoll** [comment]
I agree that the UX is unintuitive and I also tripped on it the first few times I used Grabber, but I don't have an immediate suggestion on how to improve it.

**Comment by f2d** [comment]
Some ideas:
* Flash the Downloads tab several short times, or highlight it until opened.
* Add job counter in its title, or several counters like "(1/2/3)" meaning running/finished/total, without being long and requiring more translation. Probably complicated to consistently update from various other code places, but I think it's not useless.
* Rename button captions from "Get all/this page" to "Add all/this page to Downloads tab". Simple, but needs translation.

**Comment by FichteFoll** [comment]
Yeah, renaming "Get this" to "Enqueue download" or similar sounds like a quick fix, along with an option to immediately start downloads maybe, as I personally have never needed to "edit" my download list. When I submit something for download, I have to go to the tab, find the right button to press, click that, wait for the downloads to complete, close the popup and then delete them from the download list, so they aren't downloaded again (or attempted to). That's probably not the only use case of this, since I realize you can also keep a download item to get everything with a certain tag, but that's what monitors are for now, I presume.

**Comment by ccrocker58** [comment]
Agreed. Changing the label of the "Get" options would make sense as they're misleading at the moment. That, or changing "Get <whatever>" to "Added to Download tab" after you click it.
